docs/customer_management/cp_configuration.md

```diff
@@ -29,16 +29,17 @@ ibexa:
 ## Roles and Policies
 
 You can add custom roles to your installation
-by adding them to the configuration file `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`:
+by adding them to `ibexa.site_access.config.default.corporate_accounts.roles`
+(to the default already set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`):
 
 ```yaml
 parameters:
```

Comment:
Could we actually have a full example of the override here?

Reply:
@DominikaK I'm not sure about what you're asking for.
I'll make test at lest to validate that the custom role below is added to the existing ones and replacing the whole.

Reply:
Sorry, disregard that comment, I misunderstood.